RadixWallet/Core/SharedModels/P2P/Application/IncomingMessage/P2P+RTCMessageFromPeer.swift

```diff
@@ -15,14 +15,24 @@ extension P2P {
 		}
 
 		public enum Request: Sendable, Hashable, Equatable, Decodable {
-			case dapp(P2P.Dapp.RequestUnvalidated)
+			case dapp(DappToWalletInteractionUnvalidated)
 		}
 	}
 }
 
 extension P2P.RTCMessageFromPeer.Request {
 	public init(from decoder: Decoder) throws {
```

Comment:
Do we actually need this to be generally Decodable?

Reply:
Removed Decodable